presto-main/src/main/java/com/facebook/presto/type/BooleanOperators.java

```diff
@@ -100,6 +101,13 @@ public static double castToDouble(@SqlType(StandardTypes.BOOLEAN) boolean value)
     }
 
     @ScalarOperator(CAST)
+    @SqlType(StandardTypes.REAL)
+    public static long castToReal(@SqlType(StandardTypes.BOOLEAN) boolean value)
+    {
+        return value ? floatToRawIntBits(1) : floatToRawIntBits(0);
```

Comment:
I believe `1`/`0` is fine. But I see no downside in writing `1.f`/`1.0f` and `0.f`/`0.0f`. I don't really care.